nethttp/server.go

```diff
@@ -19,6 +22,14 @@ func (w *statusCodeTracker) WriteHeader(status int) {
 	w.ResponseWriter.WriteHeader(status)
 }
 
+func (w *statusCodeTracker) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	hj, ok := w.ResponseWriter.(http.Hijacker)
+	if !ok {
+		return nil, nil, fmt.Errorf("statusCodeTracker: can't cast parent ResponseWriter to Hijacker")
```

Comment:
nit: I believe it is customary for non-parameterized errors to be declared as static vars created with `errors.New()`